napari/_qt/qt_main_window.py

```diff
@@ -677,6 +677,10 @@ def __init__(self, viewer: 'Viewer', *, show: bool = True) -> None:
         index_npe1_adapters()
 
         self._add_menus()
+        if self._qt_viewer._layers is not None:
+            add_dummy_actions(
+                self._qt_viewer._layers.model().sourceModel()._root._ctx
```

Comment:
wait why is this predicated on layers? That's a bit weird to me.

Reply:
Yep, it's v weird. Have added comment but let me know if it's still unclear...

Reply:
actually it is a great comment, thanks. 😃